ambry-store/src/test/java/com.github.ambry.store/LogSegmentTest.java

```diff
@@ -214,6 +216,16 @@ public void append(LogSegment segment, ByteBuffer buffer) throws IOException {
         assertFalse("The buffer was not completely written", buffer.hasRemaining());
       }
     });
+
+    // direct IO append
+    doAppendTest(new Appender() {
+      @Override
+      public void append(LogSegment segment, ByteBuffer buffer) throws IOException {
+        assumeTrue(Platform.isLinux());
```

Comment:
this should probably be a `if` around the `doAppendTest()`

Reply:
Changed.